tempesta_fw/procfs.h

```diff
@@ -54,15 +56,26 @@ typedef struct {
 } TfwPeerStat;
 
 /*
- * @cache_hit		- Number of Web-cache (if enabled) hits.
- * @cache_miss		- Number of misses in Web-cache (if enabled).
+ * If cache is enabled, the following stats are produced.
+ *
+ * @hits	- The number of cache hits.
+ * @misses	- The number of cache misses.
+ * @stores	- The number of cache stores.
+ * @evictions	- The number of cache evictions.
+ * @entries	- The number of cache entries.
  */
 typedef struct {
-	u64	cache_hit;
-	u64	cache_miss;
+	u64	hits;
+	u64	misses;
+	u64	stores;
+	u64	evictions;
+	u64	entries;
```

Comment:
There is no understanding how to mange the new counters, so there is no sense to add them now.